atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/output/OutputPrinter.java

```diff
@@ -18,6 +18,7 @@
 import org.slf4j.Logger;
 import org.slf4j.helpers.MessageFormatter;
 
+@SuppressWarnings("Slf4jConstantLogMessage")
```

Comment:
nit: Usually for these I prefer to explicitly state that these messages are from checkstyle, i.e. `@SuppressWarnings("checkstyle:Slf4jConstantLogMessage")`